docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -206,8 +206,14 @@ If you need to use an integration that you have previously revoked, log in again
 
 ## Why does Codacy ask for permission to create SSH keys?
 
-When you add a private repository to Codacy, Codacy uses the integration with your Git provider to create a new SSH key on the repository. Codacy then uses that SSH key every time it needs to clone the repository.
+!!! info "This section applies only to GitLab and Bitbucket"
+    For GitHub, Codacy creates an SSH key on the repository **exclusively as a fallback mechanism** when the Contents permission is missing. To avoid disruptions in the future, make sure you accept Codacy GitHub App [updated permissions](#github-cloud) on your GitHub organization.
```

Comment:
> For Github, Codacy only uses an SSH key when it's not possible to leverage the Repository Contents permission. In this case, Codacy uses a repository SSH key allow cloning and integrating with your repository. As we progress with our plan to deprecate Repository Administration permissions we will fully stop to create, use and support SSH keys. To avoid disruptions in the future, make sure you accept Codacy GitHub App [updated permissions](#github-cloud) on your GitHub organization.

`Codacy creates an SSH key on the repository` this sentence is not true yet. SSH keys are created when a repository is added, and we will only stop this behaviour when we fully deprecate the `Administration` permission.